### ISSUE TITLE
Default Board instead of upcoming cards view

### ISSUE THREAD

**Issue by aquananu** [nextcloud/deck#5130]
hi is it possible to add a field in the configuration of the deck inside the nextcloud to have a default board value which can be set by the user i.e. by selecting the board from a drop down list.

requesting this as the upcoming cards view loads very slow due to high number of board and card within that board being processed

**Comment by user8446** [comment]
I agree, as in my case I have no need for the upcoming cards view at all as I don't use due dates.

**Comment by 4jNsY6fCVqZv** [comment]
I have not yet experienced the slow loading.

Some of my tasks have due dates, but many do not.

What is the reason for the upcoming tasks being displayed as the default view so far?

Are you as a team of developers planning further views?

I would also love it if I could either select a specific board to be displayed when I open the Deck app or a kind of inbox that summarises the cards of all my personal projects in one view. With or without a due date. Or with the option to filter and that Deck remembers these filters for the next start.

**Comment by user8446** [comment]
One workaround is I have bookmarked my default board in my browser for quicker access.

**Comment by webermax** [comment]
Yes, the "filter view" loads very slow - many blocking requests to ocs/v2.php/apps/user_status/api/v1/statuses/***

Having a default board would definetively help on the short term.

On the long run decks is not usable due to general performance issues 😢
